Blockzilla/HomeView.swift

```diff
@@ -117,21 +162,55 @@ class HomeView: UIView {
         fatalError("init(coder:) has not been implemented")
     }
     
+    func showTipView() {
+        description1.isHidden = true
+        description2.isHidden = true
+        tipView.isHidden = false
+    }
+    
     func showTrackerStatsShareButton(text: String) {
         trackerStatsLabel.text = text
         trackerStatsLabel.sizeToFit()
-        trackerStatsView.isHidden = false
-        description1.isHidden = true
-        description2.isHidden = true
+        trackerStatsLabel.isHidden = false
+        trackerStatsShareButton.isHidden = false
+        shieldLogo.isHidden = false
     }
     
     func hideTrackerStatsShareButton() {
-        trackerStatsView.isHidden = true
-        description1.isHidden = false
-        description2.isHidden = false
+        shieldLogo.isHidden = true
+        trackerStatsLabel.isHidden = true
+        trackerStatsShareButton.isHidden = true
+    }
+    
+    func showTextTip(_ tip: TipManager.Tip) {
```

Comment:
Ok this might suggest that the name is bad, but this is the way to display a tip that only has text (rather than the share tip which also has a button). So it's actually displaying the entire tip...not just the text. Other suggestions?